src/kibana/directives/aria_expanded.js

```diff
@@ -0,0 +1,23 @@
+define(function (require) {
+  require('modules')
+    .get('kibana')
+    .directive('ariaExpanded', function () {
+      return {
+        restrict: 'A',
+        link: function ($scope, $elem, attrs) {
+          $elem.bind('click', function () {
+            if ($elem[0].getAttribute('aria-expanded') === 'true') {
```

Comment:
the `$elem` is a jQuery element, so you can use [.attr()](http://api.jquery.com/attr/)